cypress/e2e/PageObjects/ProfilePO.cy.js

```diff
@@ -1 +1,98 @@
 /// <reference types="cypress" />
+
+function LookingFor() {
+  return cy.get("[data-cy ='looking-for']")
+}
+
+function LookingForSecurityLink() {
+  return cy.get('#link-for-securitysettings')
+}
+
+function LookingForSecurityLinkFrench() {
+  return cy.get('#link-for-paramètresdesécurité')
+}
+
+function BackToDashboardButton() {
+  return cy.get('#back-to-dashboard-button')
+}
+
+function Eachsectionheading() {
+  return cy.get('.pb-12 >div>div')
+}
+
+function MostReqheading() {
+  return cy.get('.pb-12 >div')
+}
+
+function Cards() {
+  return cy.get('#homeContent>div')
```

Comment:
This is an example but instead of getting all divs inside the id `homeContent`, I would add a data-testId to the cards.

https://testing-library.com/docs/queries/bytestid/

Reply:
If we add a random div that is not a card inside the `#homeContent`, then this function will return more than just cards.

Reply:
I wanted to apply loop by finding multiple divs

Reply:
> If we add a random div that is not a card inside the `#homeContent`, then this function will return more than just cards.

I wanted to apply loop by finding multiple divs

Reply:
> This is an example but instead of getting all divs inside the id `homeContent`, I would add a data-testId to the cards.
> 
> https://testing-library.com/docs/queries/bytestid/

I wanted to apply loop by finding multiple divs